### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by QuangTQV** [FlagOpen/FlagEmbedding#877]
Can you tell me specifically which loss function llm embed uses and which loss function reranker uses?

**Comment by staoxiao** [comment]
Hi, @QuangTQV , embedding model and reranker model both use cross-entropy loss.
